### ISSUE TITLE
Escutcheon theme - header image

### ISSUE THREAD

**Issue by maciejpilarski** [Automattic/themes#3160]
Visually impaired user is trying to set a header image in their site using the theme “Escutcheon”.
The alt text is important for them. They have added itt, but when I set the image as the header in the customizer (besides of cropping it) the alt text is removed.

## Expected

Keeping the alt text for the header image

* User report: [#27444463-hc]
* Follow-up ticket: [#3704957-zd]

**Comment by kriskorn** [comment]
The issue is in the header.php file on line 37, where the `alt=""` does not have any PHP to load in the correct alt tag.

**Comment by akmyta** [comment]
Another report has been made here: #3730331-zd
